settings.py

```diff
@@ -46,7 +48,7 @@ def display_menu(current_menu, menu_path, selected_index, show_save_option):
         try:
             # Use red color for "Reboot" or "Shutdown"
             color = get_color("settings_sensitive" if option in sensitive_settings else "settings_default", reverse = (idx == selected_index))
-            menu_win.addstr(idx + 3, 4, f"{display_option:<{width // 2 - 2}} {display_value}".ljust(width - 8), color)
+            menu_pad.addstr(idx, 0, f"{display_option:<{width // 2 - 2}} {display_value}".ljust(width - 8), color)
         except curses.error:
             pass
```

Comment:
```suggestion
        except curses.error as e:
            logging.error(f"Failed to add line to menu_pad at idx={idx}: {e}")
```
Add this to see the error in the log

Reply:
Ah okay thanks, I see it now - lets see what's up... 

Guess I should be running this in a real IDE that will break on exceptions huh?

Reply:
Oh yes, Run and Debug in VSCode is a huge timesaver!